script.js

```diff
@@ -1,4 +1,98 @@
+  console.log(randomPlay);
+};
+
+var paperPlay = function () {
+  var randomPlay = generateRandomPlay();
+  if (randomPlay == "paper") {
+    totalDraw += 1;
+    return `The computer chose ${randomPlay}.<br>You chose paper.<br>draw<br>Now you can type "scissors" "paper" or "stone" to play another round!<br>Total computer win:${totalLose}<br>Total you win:${totalWin}<br>Total draw:${totalDraw}`;
+  }
+  if (randomPlay == "scissor") {
+    totalLose += 1;
+    var robotWinPercentage = Math.floor((totalLose / totalGamePlayed) * 100);
+    return `The computer chose ${randomPlay}.<br>You chose paper.<br>You lose!<br>Now you can type "scissors" "paper" or "stone" to play another round!<br>Total computer win:${totalLose}<br>Total you win:${totalWin}<br>Total draw:${totalDraw}<br>Robot win percentage:${robotWinPercentage}%`;
+  }
+  if (randomPlay == "stone") {
+    totalWin += 1;
+    var yourWinPercentage = Math.floor((totalWin / totalGamePlayed) * 100);
+    return `The computer chose ${randomPlay}.<br>You chose paper.<br>You Win!<br>Now you can type "scissors" "paper" or "stone" to play another round!<br>Total computer win:${totalLose}<br>Total you win:${totalWin}<br>Total draw:${totalDraw}<br>Your win percentage:${yourWinPercentage}%`;
+  }
+  console.log(randomPlay);
+};
+
+var scissorPlay = function () {
+  var randomPlay = generateRandomPlay();
+  if (randomPlay == "paper") {
+    totalWin += 1;
+    var yourWinPercentage = Math.floor((totalWin / totalGamePlayed) * 100);
+    return `The computer chose ${randomPlay}.<br>You chose scissor.<br>You Win!<br>Now you can type "scissors" "paper" or "stone" to play another round!<br>Total computer win:${totalLose}<br>Total you win:${totalWin}<br>Total draw:${totalDraw}<br>Your win percentage:${yourWinPercentage}%`;
+  }
+  if (randomPlay == "scissor") {
+    totalDraw += 1;
+    return `The computer chose ${randomPlay}.<br>You chose scissor.<br>draw!<br>Now you can type "scissors" "paper" or "stone" to play another round!<br>Total computer win:${totalLose}<br>Total you win:${totalWin}<br>Total draw:${totalDraw}`;
+  }
+  if (randomPlay == "stone") {
+    totalLose += 1;
+    var robotWinPercentage = Math.floor((totalLose / totalGamePlayed) * 100);
+    return `The computer chose ${randomPlay}.<br>You chose scissor.<br>You lose!<br>Now you can type "scissors" "paper" or "stone" to play another round!<br>Total computer win:${totalLose}<br>Total you win:${totalWin}<br>Total draw:${totalDraw}<br>Robot win percentage:${robotWinPercentage}%`;
+  }
+  console.log(randomPlay);
+};
+var gameStatus = "waiting for username";
+var username = ``;
 var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+  if (gameStatus == `waiting for username`) {
+    username = input;
+    gameStatus = `scissor paper stone`;
+    return `hello ${username}, please input "scissor, "paper", or "stone"`;
+  } else if (gameStatus == `scissor paper stone`)
+    return paperScissorStone(input);
```

Comment:
good job breaking up your code into smaller helper functions. I like that your main function was easy to read and succinct